src/view/ErrorFallBack.tsx

```diff
@@ -0,0 +1,226 @@
+import {
+  Heading,
+  Text,
+  UnorderedList,
+  ListItem,
+  Collapse,
+  Box,
+  ChakraProvider
+} from '@chakra-ui/react';
+import { useState } from 'react';
+
+function ErrorFallback() {
+  const [openItem, setOpenItem] = useState<string | null>(null);
+
+  const handleToggle = (item: string) => {
+    setOpenItem(openItem === item ? null : item);
+  };
+
+  return (
+    <ChakraProvider>
+      <Box border="5px solid rgba(0,0,0,0.2)" p={4}>
+        <div>
+          <Heading size="xl">Something Went Wrong...</Heading>
+          <Heading size="md" mt={4} color="magenta.500">
+            Troubleshooting Guide
+          </Heading>
+          <Text>
+            If you&apos;re experiencing issues, follow the steps below to
+            troubleshoot and resolve common problems:
+          </Text>
+          <UnorderedList>
+            <ListItem>
```

Comment:
Can these list items be abstracted into a reusable component? Seems that some duplicate code could be reduced